features/openhab-addons/src/main/feature/feature.xml

```diff
@@ -173,6 +173,11 @@
         <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.lgtvserial/${project.version}</bundle>
     </feature>
 
+    <feature name="openhab-binding-lgwebos" description="LG webOS Binding" version="${project.version}">
```

Comment:
Note that the Apache HTTP client libs are no longer part of a default installation. As your binding requires them, you need to explicitly add them here (like it was done [here](https://github.com/openhab/openhab2-addons/commit/b9d15820cf9b71b9ab46693ac21d6e6d7958758d)).

Reply:
Done